vault/logical_system.go

```diff
@@ -33,6 +36,7 @@ import (
 	"github.com/hashicorp/vault/logical"
 	"github.com/hashicorp/vault/logical/framework"
 	"github.com/mitchellh/mapstructure"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I'd strongly prefer to not have this be a direct dependency of Vault's core.

Reply:
Alright